resolver/min_conns_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	"context"
 	"testing"
 
-	. "github.com/bufbuild/httplb/resolver"
+	. "github.com/bufbuild/httplb/resolver" //nolint:revive
```

Comment:
Why are we ignoring this? What was the linter complaint? I think this warrants a comment. (Here and below.)

Reply:
`dot-imports: should not use dot imports (revive)`